docs/faq.rst

```diff
@@ -0,0 +1,27 @@
+FAQ
+===
+
+.. _fallback-endpoint:
+
+Can I register an endpoint that accepts all requests?
+-----------------------------------------------------
+
+The fallback endpoint is the endpoint called when an unrecognized request is
+received. By default, the fallback endpoint simply returns a
+``BadRequestError`` to the caller. This behavior may be overriden by
+registering a raw endpoint with ``TChannel.FALLBACK``.
+
+.. code-block:: python
+
+    from tchannel import TChannel
+
+    server = TChannel(name='myservice')
+
+    @server.register('raw', TChannel.FALLBACK)
```

Comment:
No. Raw counts as all schemes. Basically, the scheme equality check is skipped if the handler is a fallback.